modules/serverless-kn-container.adoc

```diff
@@ -0,0 +1,60 @@
+[id="serverless-kn-container_{context}"]
+= Knative client multi-container support
+
+You can use the `kn container add` command to print YAML container spec to standard output. This command is useful for multi-container use cases because it can be used along with other standard `kn` flags to create definitions. The `kn container add` command accepts all container-related flags that are supported for use with the `kn service create` command. The `kn container add` command can also be chained through Unix pipe symbols (`|`) to create multiple containers at once.
```

Comment:
Here just minor wording suggestion.

```suggestion
You can use the `kn container add` command to print YAML container spec to standard output. This command is useful for multi-container use cases because it can be used along with other standard `kn` flags to create definitions. The `kn container add` command accepts all container-related flags that are supported for use with the `kn service create` command. The `kn container add` command can also be chained through Unix pipes (`|`) to create multiple containers definitions at once.
```